tests/scenarios/vite-app-test.ts

```diff
@@ -3,8 +3,10 @@ import type { PreparedApp } from 'scenario-tester';
 import QUnit from 'qunit';
 
 const { module: Qmodule, test } = QUnit;
+const isWin = process.platform === 'win32';
```

Comment:
so it's a nice solution but I'm not 100% sure it will work 🤔 it might, but it might also not be a supported scenario-tester technique

We probably want to filter over here https://github.com/embroider-build/embroider/blob/main/test-packages/support/suite-setup-util.ts#L31 because that's where we filter things for windows already.

It does beg the question: are we not intending to support Vite on windows? is there something specific that is causing a problem right now?

Reply:
I want to also support windows, but currently it fails.
I do not want to make this pr bigger than it is.
Let's try to get this to work good on unix. Afterwards i can try to fix it on windows. I do mostly work on windows anyway:)

Reply:
I just realised I didn't try to fix thr path from process.cwd(). Will give it another try